src/core/interfaces.ts

```diff
@@ -150,18 +158,22 @@ export interface INamespace {
 
 export interface ITransaction {
   created: number;
-  hash: string;
   id: string;
-  protocolId?: string;
-  sequence: number;
-  status: TXStatus;
-  info?: IEthTransactionInfo;
-  subject: {
-    signer: string;
-    namespace: string;
+  namespace: string;
+  type: TransactionType;
+}
+
+export interface ITransactionStatus {
+  status: string;
```

Comment:
status (here and in the details) is an enum of Pending/Failed/Succeeded, so might be useful to define that